lib/configure.js

```diff
@@ -362,7 +362,7 @@ PythonFinder.prototype = {
   log: logWithPrefix(log, 'find Python'),
   argsExecutable: ['-c', 'import sys; print(sys.executable);'],
   argsVersion: ['-c', 'import sys; print("%s.%s.%s" % sys.version_info[:3]);'],
-  semverRange: '>=2.6.0 <3.0.0',
+  semverRange: '>=2.7.0 <3.8.0',
```

Comment:
Do we need an upper bound on the semver check anymore? It was there before to exclude Python 3.

Reply:
Now it is __semverRange: '>=2.7.0'__